docs/content/docs/how-to-guides/13_deployEnbuildQuickstartEnbuildCLI.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Deploying ENBUILD using ENBUILD CLI"
+description: "Steps to Configure ENBUILD CLI and deploy ENBUILD using ENBUILD CLI"
+summary: "Configure ENBUILD CLI and deploy ENBUILD using ENBUILD CLI"
+draft: false
+menu:
+  docs:
+    parent: "docs/how-to-guides/deploying-enbuild-for-local-testing/"
+    identifier: "configureEnbuildCLI"
+weight: 202
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Configure ENBUILD CLI
+
+Follow these step-by-step instructions to configure ENBUILD CLI.
+
+### Prerequisites
+
+Make sure you install the following dependencies.
+
+1. [Docker](https://docs.docker.com/engine/install/)
+    - Install docker by following these [steps](https://docs.docker.com/engine/install/).
+    - Make sure that docker engine is running before going using the Enbuild CLI.
+
+2. [Golang](https://go.dev/)
+    - Install Go programming language by following these [steps](https://go.dev/doc/install).
+
+3. [yq cli](https://mikefarah.gitbook.io/yq)
+    - Install yq cli following these [steps](https://github.com/mikefarah/yq/#install).
+    - Enbuild cli is using it internally for creating bigbang catalog template values file.
+
+**Note:** Additional dependencies such as k3d, Helm, jq, grep, sed, curl, and iproute2 will be checked and installed by the ENBUILD CLI when executing the commands.
+
+
+### Configuration
+
+1. Clone the [Enbuild repository](https://github.com/vivsoftorg/enbuild.git)
+
+    ``` bash
+    git clone https://github.com/vivsoftorg/enbuild.git
+    ```
+
+2. Change your directory to `cli` in the enbuild repository
+
+    ``` bash
+    cd <path-to-the-above-cloned-enbuild-repository>/enbuild/cli
+    ```
+
+3. Run the below command to build the `enbuild` cli
+
+    ```bash
+    go build
+    ```
```

Comment:
we dont need document on building it, users can download the prebuilt binary from the Release page depending on their platform https://github.com/vivsoftorg/enbuild/releases/tag/v0.0.11

Reply:
Updated.